packages/cloudflare/package.json

```diff
@@ -45,14 +45,14 @@
     "tiny-glob": "^0.2.9"
   },
   "peerDependencies": {
-    "astro": "^4.0.0"
+    "astro": "^4.2.0"
```

Comment:
I'm curious, what change in the adapter makes it incompatible with `4.0`? I saw the change to `render`, but don't see how that requires an upgrade here. Or is this more of a nudge to keep users on current versions?

Reply:
Good point. This bump **isn't** needed if the adapters don't use the new APIs provided by `astro/app`

Reply:
You refer to https://github.com/withastro/astro/pull/9661, right? Is there anything else that's new in 4.2 that's useful to adapters?

Reply:
Those quality-of-life additions are definitely nice and useful, but I'm worried about raising the peer dependency so fast here. What about we wait for a couple weeks with using them, to give users time to upgrade?

Reply:
Yeah I don't mind waiting a bit :)

Reply:
The point why I bumped the peer dependency to `4.2`, was that we would already be ready for the new features if we use them in a couple of weeks. Otherwise we would need to do another major release in two weeks or so. If that is fine for everyone, we can also take the approach of two major releases :)

Reply:
Oh no, also don't want us to do two major releases when it could be one. If we don't bump the peer dependency here, why do we need to cut a major release?

Reply:
In theory it is a breaking change if we bump peer dependency versions, the user would need to do manual changes to keep the site compatible. We always handled it in the past like this, IIRC.

On the other hand, we only have to do another major release once we use the new features of 4.2, and we don't know yet if that would be weeks or months.

Reply:
Yeah that make sense, and is fine for me. I'll set the peer dependencies for  the netlify adapter to `^4.0.0`, and once we start using the new features in the netlify adapter we can cut another major release.

_For Cloudflare I'll already use this release, because I'm planning to add the new features next week._